astroimtools/cutout_tools.py

```diff
@@ -16,11 +16,267 @@
 from astropy import log
 from astropy.coordinates import SkyCoord
 from astropy.io import fits
+from astropy.io.fits import PrimaryHDU, ImageHDU, CompImageHDU
 from astropy.nddata.utils import (Cutout2D, NoOverlapError)
-from astropy.table import QTable
+from astropy.table import QTable, Table
 from astropy.wcs import WCS, NoConvergence
+from astropy.wcs.utils import proj_plane_pixel_scales
 
-__all__ = ['make_cutouts', 'show_cutout_with_slit']
+__all__ = ['make_cutouts', 'show_cutout_with_slit', 'cutout_tool']
+
+
+def cutout_tool(image, catalog, image_ext=0, origin=0,
+                to_fits=False, output_dir="cutouts", overwrite=False,
+                catalog_format='ascii.ecsv', delimiter=None,
+                suppress_rotation=False, verbose=True):
+    """
+    Make cutouts from a 2D image and write them to FITS files.
+
+    The input Catalog must have the following columns with unit info, where applicable:
+
+        * ``'id'`` - ID string; no unit necessary.
+        * ``'ra'`` or ``'x'``- RA (e.g., in degrees) or pixel x position (in units.pix).
+        * ``'dec'`` or ``'y'`` - DEC (e.g., in degrees) or pixel y position (in units.pix).
+        * ``'cutout_width'`` - Cutout width (e.g., in arcsec, pix).
+        * ``'cutout_height'`` - Cutout height (e.g., in arcsec, pix).
+    Optional columns:
+        * ``'cutout_pa'`` - Cutout angle (e.g., in degrees). This is only
```

Comment:
Do you mean "for example" or is degrees the only option?  If just degrees I would be explicit. If it can be in radians, I would just say "(degrees or radians)".